### PR TITLE
Update docs for v1.0.7

### PR DESCRIPTION
**Pull request by prakshalj0512**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,12 @@ In case you want to do development work on the project
     * v1.0.6
         * Added feature to poll the health URL to ensure the Scheduler is Healthy
         * RELEASE
+    * v1.0.7
+        * Disable SSL Verify for Airflow API Request
+        * RELEASE
+    * v1.0.8
```
Comment (rssanders3) [inline README.md:78]: Remove this version

**Review by rssanders3 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -33,3 +33,11 @@ Includes base functionality to control and check if the Scheduler is running. Su
 
 * Added feature to poll the health URL to ensure the Scheduler is Healthy
 * Updating Documentation in README
+
+### [v1.0.7](https://github.com/teamclairvoyant/airflow-scheduler-failover-controller/tree/v1.0.7)
+
+* Disabled SSL verify for Airflow API Request
+
+### [v1.0.8](https://github.com/teamclairvoyant/airflow-scheduler-failover-controller/tree/v1.0.8)
```
Comment (rssanders3) [inline CHANGELOG.md:41]: Remove here as well

**Review by rssanders3 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (prakshalj0512) [reply CHANGELOG.md]: doen

**Review by rssanders3 — APPROVED** [review]
LGTM

**Pull request merged**